nvflare/app_common/workflows/statistics_controller.py

```diff
@@ -409,14 +409,16 @@ def _combine_all_statistics(self):
                             hist: Histogram = self.client_statistics[statistic][client][ds][feature_name]
                             buckets = StatisticsController._apply_histogram_precision(hist.bins, self.precision)
                             result[feature_name][statistic][client][ds] = buckets
-                        elif statistic == StC.STATS_PERCENTILE:
-                            percentiles = self.client_statistics[statistic][client][ds][feature_name][
-                                StC.STATS_PERCENTILES_KEY
-                            ]
-                            formatted_percentiles = {}
-                            for p in percentiles:
-                                formatted_percentiles[p] = round(percentiles.get(p), self.precision)
-                            result[feature_name][statistic][client][ds] = formatted_percentiles
+                        elif statistic == StC.STATS_QUANTILE:
+
+                            if not check_fastdigest_installed():
```

Comment:
I think we should not do optional import in app_common.

Suggest either:
(1) move this file to app_opt
(2) create a new quantile_statistics_controller.py in app_opt and these logic should be moved there